manifests/config.pp

```diff
@@ -41,7 +41,7 @@
   foreman_proxy::settings_file { 'tftp': }
   foreman_proxy::settings_file { 'realm': }
 
-  if $foreman_proxy::puppetca or $foreman_proxy::puppetrun {
+  if $foreman_proxy::puppetca or $foreman_proxy::puppetrun or tagged('foreman_proxy::plugin::salt') {
```

Comment:
That's only going to work if the salt class was parsed before this one, it's not reliable.